web/themes/custom/votegov/src/sass/uswds-overrides/usa-alert.scss

```diff
@@ -0,0 +1,20 @@
+@use "uswds-core" as *;
```

Comment:
<img width="1506" alt="Screenshot 2024-11-17 at 9 36 17 PM" src="https://github.com/user-attachments/assets/a76c5a58-f013-40de-ae60-8431ace0fdbc">

here is a side by side for EN and AR